packages/video_player/video_player_avfoundation/ios/Classes/AVAssetTrack+Utils.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <AVFoundation/AVFoundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+// Fix the transform for the track.
+// Each fix case corresponding to `UIImage.Orientation`, with 8 cases in total.
```

Comment:
This is an implementation detail, and doesn't belong in the declaration comment. The declaration comment should explain what a method does and what it's for, not exactly how it's implemented.

Reply:
Comments are all reorganized, PTAL.